### ISSUE TITLE
QML Line Numbers

### ISSUE THREAD

**Issue by juliandoerner** [limetext/lime-qml#1]
Reopening old Issue in relevant Repo
https://github.com/limetext/lime/issues/516

**Comment by ricochet1k** [comment]
Basic line numbers added by 883fb79374e825e8c516b3a58eccde287de0dfbf
Full gutter support is waiting on backend.

**Comment by zoli** [comment]
Regarding to [color schemes docs](http://docs.sublimetext.info/en/latest/reference/color_schemes.html#gutter) you mean `gutter` and `gutterForeground`?

**Comment by ricochet1k** [comment]
Oh, yeah, I need to work on respecting user settings.

I was referring to gutter icons.

**Comment by zoli** [comment]
backend now exports global settings https://github.com/limetext/backend/commit/d3fe730962b9787ceac7f09a07784e8dca9c6335, also sublime has implemented this to https://github.com/limetext/sublime/commit/d185d6ac6ea0a81425539cfd8529128bffd9ea11.

**Comment by zoli** [comment]
This is closed by 883fb79374e825e8c516b3a58eccde287de0dfbf, we should open another issue for gutter support.
